### DISCUSSION TITLE
how to install plustoolkit on raspberry pi 3?

### DISCUSSION THREAD

**Discussion by farhannizam** [PlusToolkit/PlusLib#930] [Q&A]
how to install plustoolkit on raspberry pi 3? i try many times but it say cant install

**Comment by lassoan** [comment]
We only provide binaries for Windows, so you can either run Plus on a [Windows compute stick](https://www.amazon.ca/s?k=compute+stick&i=electronics&crid=3ORDRMLHYGQBW&sprefix=compute+stick%2Celectronics%2C89&ref=nb_sb_noss_1), or you can build and run Plus on the Raspberry Pi. If you need GUI then the Windows stick is easier, because everything should just work; while building and running Qt on an embedded linux system can be tricky (because OpenGL support is somewhat limited).

What Plus devices you would like to use on the mini-computer?

**Reply by farhannizam** [reply]
this is my first time running project on raspberry pi. i want to make like this project
![Capture](https://user-images.githubusercontent.com/101652334/161906692-4ade86c8-d6f3-4420-92e6-d9a7ee48118a.PNG)

they using plustoolkit but they dont mention what devices that they use or maybe i dont know what kind device isnt .

**Reply by lassoan** [reply]
The Raspberry Pi is a very underpowered device compared to the slowest, smallest laptop or NUC PC. I would not recommend considering using a Raspberry Pi in general, but you may consider it if you need to operate on batteries and you don't need to have any GUI (for example, just process network or serial input and provide network output).

**Comment by adamrankin** [comment]
I have successfully build Plus for the Pi (and other arm64). But I've never tried to build the GUI portion.
